package.json

```diff
@@ -3,28 +3,24 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev:vite": "yarn start:vite",
-    "start:vite": "yarn generate-icons:vite && vite --host",
-    "build:vite": "yarn generate-icons:vite && tsc -b && vite build",
     "dev": "yarn start",
-    "start": "craco start",
-    "build": "yarn generate-icons:webpack && craco build --config craco.config.cjs",
-    "build:stats": "craco build --config craco.stats.config.cjs",
-    "test": "craco test",
-    "test:vite": "yarn generate-icons:vite && vitest run --changed --config ./vitest.config.mjs",
-    "test:watch:vite": "yarn generate-icons:vite && vitest watch --changed --config ./vitest.config.mjs",
-    "test:ci:vite": "yarn generate-icons:vite && vitest run --config ./vitest.config.mjs --coverage",
+    "dev:vite": "yarn start",
```

Comment:
Per our chat, the commands with suffix `:vite` ones with get removed in a separate phase after Infra does stuff